CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 ## Feedback
 
 The report of issues and the proposal of new features is very welcome.
```

Comment:
This line sounds backwards -- "Issue reports and feature proposals are very welcome."